profile/profile.yaml

```diff
@@ -0,0 +1,119 @@
+      - "JDBC App: Curabitur laoreet tristique leo, eget suscipit nisi. Sed in sodales ex. Maecenas vitae tincidunt dui, et eleifend quam."
+      - "Grep App: Curabitur laoreet tristique leo, eget suscipit nisi. Sed in sodales ex. Maecenas vitae tincidunt dui, et eleifend quam."
+  - name: "Springboot App"
+    git_url: "/springboot"
+    description: ""
+  - name: "Python Data Analytics"
+    git_url: "/python_data_anlytics"
+    description: ""
+  - name: "Hadoop"
+    git_url: "/hadoop"
+    description: ""
+  - name: "Spark"
+    git_url: "/spark"
+    description: ""
+  - name: "Cloud/DevOps"
+    git_url: "/cloud_devops"
+    description: ""
+
+# List of personal or academic projects
+# Feel free to add more projects
+highlighted_projects:
+    #project name
+  - name: "Tutoring Web App"
+    #URL is optional
+    git_url: "https://github.com/pkandic4/TutoringApplication"
+    # The project description must start with an action verb (https://bit.ly/3guO98u)
+    description: Designed a web tutoring app. The app dynamically displays a range of subjects for which tutors are available. The login system stores and retrieves account information using SQL. Active display elements use JavaScript.
+  - name: "Java Car Dealership Simulator"
+    git_url: "https://github.com/pkandic4/CarDealershipSimulator"
+    description: Programmed a fully functional car dealership simulator in Java. A list of cars - including makes, prices, and other information - is imported from a text file. Available commands include the ability to buy cars, return them, and to filter the list based on certain criteria. There are also advanced commands, including filtering the sales team based on their transactions. Error handling is implemented to ensure the program operates in a stable manner.
+
+#List of professional experience
+#The job description allows hiring managers to understand your job and the technologies you used(e.g. programming languages, libraries, algorithms, hardware, tools, etc.).
+#The description must be more than 25 words. If your experience is not software related, please focus on soft skills and achievements (e.g. team collaboration, problem-solving, improvements, communication).
+professional_experience:
+  #Write your Jarvis role as a professional experience rather than training.
+  - title: "Software Developer"
+    company: "Jarvis"
+    duration: "2023-present"
+    # The project description must start with an action verb (https://bit.ly/3guO98u)
+    description: "Working alone and with a team to develop software."
+  - title: "Data Engineer"
+    company: "Fiix Software"
+    duration: "2021"
+    description: "Developed web scraping software, and then populated databases with the gathered data. The scraper I developed executed 35% faster than its predecessor, with 40% fewer lines of code. In addition, I developed an automated script to transfer data into company databases."
```

Comment:
Developed web scraping software with Python (pandas, http), Curl, etc.
Techs
Automation
Results (faster, fewer code) + biz value( provide data to the biz team quicker)